pkg/cloud/aws/athenaintegration.go

```diff
@@ -94,8 +94,9 @@ func (ai *AthenaIntegration) GetCloudCost(start, end time.Time) (*opencost.Cloud
 	// of columns to query.
 	for column := range allColumns {
 		if strings.HasPrefix(column, LabelColumnPrefix) {
-			groupByColumns = append(groupByColumns, column)
-			aqi.TagColumns = append(aqi.TagColumns, column)
+			quotedTag := fmt.Sprintf("\"%s\"", column)
```

Comment:
You could make this a little cleaner like this I believe:
```suggestion
			quotedTag := fmt.Sprintf(`"%s"`, column)
```